src/core/client/framework/lib/errors/reporter/sentry/sentry.tsx

```diff
@@ -67,7 +67,8 @@ export class SentryErrorReporter implements ErrorReporter {
       release: `coral@${process.env.TALK_VERSION}`,
       debug: Boolean(options.offlineDebug),
       transport: options.offlineDebug ? FakeDebugTransport : undefined,
-      whitelistUrls,
+      allowUrls: whitelistUrls,
+      sampleRate: 0.2,
```

Comment:
This will set a sample rate of 20% for all events going to Sentry. My recommendation was that we set up a sampling rate only for events that have no domain tag value set since that's what we were seeing with these events we want to filter. https://docs.sentry.io/platforms/javascript/configuration/sampling/#setting-a-sampling-function

But we could also try a blanket sampling rate and see if we feel that's working well since it could limit any future spikes.

Reply:
yeah i'd like to start out quite conservatively just because we're sharing sentry with other teams. If it all seems manageable we can be more specific in the future.